{
	"documenttitle": {
		"text": "Red Hat Security Advisory: unbound security, bug fix, and enhancement update",
		"lang": "en"
	},
	"DocumentType": "Security Advisory",
	"documentpublisher": {
		"type": "Vendor",
		"ContactDetails": "secalert@redhat.com",
		"IssuingAuthority": "Red Hat Product Security"
	},
	"documenttracking": {
		"identification": {
			"ID": "RHSA-2021:1853"
		},
		"Status": "Final",
		"Version": "1",
		"revisionhistory": {
			"revision": {
				"Number": "1",
				"Date": "2021-05-18T14:39:00Z",
				"Description": "Current version"
			}
		},
		"InitialReleaseDate": "2021-05-18T14:39:00Z",
		"CurrentReleaseDate": "2021-05-18T14:39:00Z",
		"generator": {
			"Engine": "Red Hat SDEngine 3.0.0",
			"Date": "2021-08-23T05:00:00Z"
		}
	},
	"documentnotes": {
		"note": [
			{
				"text": "\nAn update for unbound is now available for Red Hat Enterprise Linux 8.\n\nRed Hat Product Security has rated this update as having a security impact of Moderate. A Common Vulnerability Scoring System (CVSS) base score, which gives a detailed severity rating, is available for each vulnerability from the CVE link(s) in the References section.\n    ",
				"title": "Topic",
				"type": "Summary",
				"ordinal": "1",
				"lang": "en"
			},
			{
				"text": "\nThe unbound packages provide a validating, recursive, and caching DNS or DNSSEC resolver. \n\nSecurity Fix(es):\n\n* unbound: integer overflow in the regional allocator via regional_alloc (CVE-2019-25032)\n\n* unbound: integer overflow in sldns_str2wire_dname_buf_origin can lead to an out-of-bounds write (CVE-2019-25034)\n\n* unbound: out-of-bounds write in sldns_bget_token_par (CVE-2019-25035)\n\n* unbound: assertion failure and denial of service in synth_cname (CVE-2019-25036)\n\n* unbound: assertion failure and denial of service in dname_pkt_copy via an invalid packet (CVE-2019-25037)\n\n* unbound: integer overflow in a size calculation in dnscrypt/dnscrypt.c (CVE-2019-25038)\n\n* unbound: integer overflow in a size calculation in respip/respip.c (CVE-2019-25039)\n\n* unbound: infinite loop via a compressed name in dname_pkt_copy (CVE-2019-25040)\n\n* unbound: assertion failure via a compressed name in dname_pkt_copy (CVE-2019-25041)\n\n* unbound: out-of-bounds write via a compressed name in rdata_copy (CVE-2019-25042)\n\n* unbound: symbolic link traversal when writing PID file (CVE-2020-28935)\n\nFor more details about the security issue(s), including the impact, a CVSS score, acknowledgments, and other related information, refer to the CVE page(s) listed in the References section.\n\nAdditional Changes:\n\nFor detailed information on changes in this release, see the Red Hat Enterprise Linux 8.4 Release Notes linked from the References section.\n    ",
				"title": "Details",
				"type": "General",
				"ordinal": "2",
				"lang": "en"
			},
			{
				"text": "This content is licensed under the Creative Commons Attribution 4.0 International License (https://creativecommons.org/licenses/by/4.0/). If you distribute this content, or a modified version of it, you must provide attribution to Red Hat Inc. and provide a link to the original.",
				"title": "Terms of Use",
				"type": "Legal Disclaimer",
				"ordinal": "3",
				"lang": "en"
			}
		]
	},
	"documentdistribution": {
		"text": "Copyright Â© 2021 Red Hat, Inc. All rights reserved.",
		"lang": "en"
	},
	"aggregateseverity": {
		"text": "Moderate",
		"namespace": "https://access.redhat.com/security/updates/classification/"
	},
	"documentreferences": {
		"reference": [
			{
				"type": "Self",
				"URL": "https://access.redhat.com/errata/RHSA-2021:1853",
				"Description": "https://access.redhat.com/errata/RHSA-2021:1853"
			},
			{
				"type": "External",
				"URL": "https://access.redhat.com/security/updates/classification/#moderate",
				"Description": "https://access.redhat.com/security/updates/classification/#moderate"
			},
			{
				"type": "External",
				"URL": "https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/8.4_release_notes/",
				"Description": "https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/8.4_release_notes/"
			}
		]
	},
	"vulnerability": [
		{
			"ordinal": "1",
			"xmlns": "http://www.icasi.org/CVRF/schema/vuln/1.1",
			"notes": {
				"note": {
					"text": "unbound: integer overflow in the regional allocator via regional_alloc",
					"title": "Vulnerability Description",
					"type": "General",
					"ordinal": "1",
					"lang": "en"
				}
			},
			"ReleaseDate": "2019-12-11T00:00:00Z",
			"involvements": {
				"involvement": {
					"party": "Vendor",
					"status": "Completed"
				}
			},
			"CVE": "CVE-2019-25032",
			"remediations": {
				"remediation": {
					"type": "Vendor Fix",
					"description": {
						"text": "\nFor details on how to apply this update, which includes the changes described in this advisory, refer to:\n\nhttps://access.redhat.com/articles/11258\n    ",
						"lang": "en"
					},
					"URL": "https://access.redhat.com/errata/RHSA-2021:1853"
				}
			},
			"references": {
				"reference": [
					{
						"URL": "https://access.redhat.com/security/cve/CVE-2019-25032",
						"Description": "CVE-2019-25032"
					},
					{
						"URL": "https://bugzilla.redhat.com/show_bug.cgi?id=1954772",
						"Description": "bz#1954772: CVE-2019-25032 unbound: integer overflow in the regional allocator via regional_alloc"
					}
				]
			},
			"threats": {
				"threat": {
					"type": "Impact",
					"Description": "Moderate"
				}
			},
			"DiscoveryDate": "2021-04-27T00:00:00Z",
			"productstatuses": {
				"status": {
					"type": "Fixed",
					"ProductID": [
						"AppStream-8.4.0.GA:unbound-0:1.7.3-15.el8"
					]
				}
			},
			"acknowledgments": {
				"acknowledgment": {
					"Description": ""
				}
			},
			"cvssscoresets": {
				"scoreset": {
					"BaseScore": "",
					"Vector": ""
				}
			}
		},
		{
			"ordinal": "2",
			"xmlns": "http://www.icasi.org/CVRF/schema/vuln/1.1",
			"notes": {
				"note": {
					"text": "unbound: integer overflow in sldns_str2wire_dname_buf_origin can lead to an out-of-bounds write",
					"title": "Vulnerability Description",
					"type": "General",
					"ordinal": "1",
					"lang": "en"
				}
			},
			"ReleaseDate": "2019-12-11T00:00:00Z",
			"involvements": {
				"involvement": {
					"party": "Vendor",
					"status": "Completed"
				}
			},
			"CVE": "CVE-2019-25034",
			"remediations": {
				"remediation": {
					"type": "Vendor Fix",
					"description": {
						"text": "\nFor details on how to apply this update, which includes the changes described in this advisory, refer to:\n\nhttps://access.redhat.com/articles/11258\n    ",
						"lang": "en"
					},
					"URL": "https://access.redhat.com/errata/RHSA-2021:1853"
				}
			},
			"references": {
				"reference": [
					{
						"URL": "https://access.redhat.com/security/cve/CVE-2019-25034",
						"Description": "CVE-2019-25034"
					},
					{
						"URL": "https://bugzilla.redhat.com/show_bug.cgi?id=1954778",
						"Description": "bz#1954778: CVE-2019-25034 unbound: integer overflow in sldns_str2wire_dname_buf_origin can lead to an out-of-bounds write"
					}
				]
			},
			"threats": {
				"threat": {
					"type": "Impact",
					"Description": "Moderate"
				}
			},
			"DiscoveryDate": "2021-04-27T00:00:00Z",
			"productstatuses": {
				"status": {
					"type": "Fixed",
					"ProductID": [
						"AppStream-8.4.0.GA:unbound-0:1.7.3-15.el8"
					]
				}
			},
			"acknowledgments": {
				"acknowledgment": {
					"Description": ""
				}
			},
			"cvssscoresets": {
				"scoreset": {
					"BaseScore": "",
					"Vector": ""
				}
			}
		},
		{
			"ordinal": "3",
			"xmlns": "http://www.icasi.org/CVRF/schema/vuln/1.1",
			"notes": {
				"note": {
					"text": "unbound: out-of-bounds write in sldns_bget_token_par",
					"title": "Vulnerability Description",
					"type": "General",
					"ordinal": "1",
					"lang": "en"
				}
			},
			"ReleaseDate": "2019-12-11T00:00:00Z",
			"involvements": {
				"involvement": {
					"party": "Vendor",
					"status": "Completed"
				}
			},
			"CVE": "CVE-2019-25035",
			"remediations": {
				"remediation": {
					"type": "Vendor Fix",
					"description": {
						"text": "\nFor details on how to apply this update, which includes the changes described in this advisory, refer to:\n\nhttps://access.redhat.com/articles/11258\n    ",
						"lang": "en"
					},
					"URL": "https://access.redhat.com/errata/RHSA-2021:1853"
				}
			},
			"references": {
				"reference": [
					{
						"URL": "https://access.redhat.com/security/cve/CVE-2019-25035",
						"Description": "CVE-2019-25035"
					},
					{
						"URL": "https://bugzilla.redhat.com/show_bug.cgi?id=1954780",
						"Description": "bz#1954780: CVE-2019-25035 unbound: out-of-bounds write in sldns_bget_token_par"
					}
				]
			},
			"threats": {
				"threat": {
					"type": "Impact",
					"Description": "Moderate"
				}
			},
			"DiscoveryDate": "2021-04-27T00:00:00Z",
			"productstatuses": {
				"status": {
					"type": "Fixed",
					"ProductID": [
						"AppStream-8.4.0.GA:unbound-0:1.7.3-15.el8"
					]
				}
			},
			"acknowledgments": {
				"acknowledgment": {
					"Description": ""
				}
			},
			"cvssscoresets": {
				"scoreset": {
					"BaseScore": "",
					"Vector": ""
				}
			}
		},
		{
			"ordinal": "4",
			"xmlns": "http://www.icasi.org/CVRF/schema/vuln/1.1",
			"notes": {
				"note": {
					"text": "unbound: assertion failure and denial of service in synth_cname",
					"title": "Vulnerability Description",
					"type": "General",
					"ordinal": "1",
					"lang": "en"
				}
			},
			"ReleaseDate": "2019-12-11T00:00:00Z",
			"involvements": {
				"involvement": {
					"party": "Vendor",
					"status": "Completed"
				}
			},
			"CVE": "CVE-2019-25036",
			"remediations": {
				"remediation": {
					"type": "Vendor Fix",
					"description": {
						"text": "\nFor details on how to apply this update, which includes the changes described in this advisory, refer to:\n\nhttps://access.redhat.com/articles/11258\n    ",
						"lang": "en"
					},
					"URL": "https://access.redhat.com/errata/RHSA-2021:1853"
				}
			},
			"references": {
				"reference": [
					{
						"URL": "https://access.redhat.com/security/cve/CVE-2019-25036",
						"Description": "CVE-2019-25036"
					},
					{
						"URL": "https://bugzilla.redhat.com/show_bug.cgi?id=1954782",
						"Description": "bz#1954782: CVE-2019-25036 unbound: assertion failure and denial of service in synth_cname"
					}
				]
			},
			"threats": {
				"threat": {
					"type": "Impact",
					"Description": "Moderate"
				}
			},
			"DiscoveryDate": "2021-04-27T00:00:00Z",
			"productstatuses": {
				"status": {
					"type": "Fixed",
					"ProductID": [
						"AppStream-8.4.0.GA:unbound-0:1.7.3-15.el8"
					]
				}
			},
			"acknowledgments": {
				"acknowledgment": {
					"Description": ""
				}
			},
			"cvssscoresets": {
				"scoreset": {
					"BaseScore": "",
					"Vector": ""
				}
			}
		},
		{
			"ordinal": "5",
			"xmlns": "http://www.icasi.org/CVRF/schema/vuln/1.1",
			"notes": {
				"note": {
					"text": "unbound: assertion failure and denial of service in dname_pkt_copy via an invalid packet",
					"title": "Vulnerability Description",
					"type": "General",
					"ordinal": "1",
					"lang": "en"
				}
			},
			"ReleaseDate": "2019-12-11T00:00:00Z",
			"involvements": {
				"involvement": {
					"party": "Vendor",
					"status": "Completed"
				}
			},
			"CVE": "CVE-2019-25037",
			"remediations": {
				"remediation": {
					"type": "Vendor Fix",
					"description": {
						"text": "\nFor details on how to apply this update, which includes the changes described in this advisory, refer to:\n\nhttps://access.redhat.com/articles/11258\n    ",
						"lang": "en"
					},
					"URL": "https://access.redhat.com/errata/RHSA-2021:1853"
				}
			},
			"references": {
				"reference": [
					{
						"URL": "https://access.redhat.com/security/cve/CVE-2019-25037",
						"Description": "CVE-2019-25037"
					},
					{
						"URL": "https://bugzilla.redhat.com/show_bug.cgi?id=1954794",
						"Description": "bz#1954794: CVE-2019-25037 unbound: assertion failure and denial of service in dname_pkt_copy via an invalid packet"
					}
				]
			},
			"threats": {
				"threat": {
					"type": "Impact",
					"Description": "Moderate"
				}
			},
			"DiscoveryDate": "2021-04-27T00:00:00Z",
			"productstatuses": {
				"status": {
					"type": "Fixed",
					"ProductID": [
						"AppStream-8.4.0.GA:unbound-0:1.7.3-15.el8"
					]
				}
			},
			"acknowledgments": {
				"acknowledgment": {
					"Description": ""
				}
			},
			"cvssscoresets": {
				"scoreset": {
					"BaseScore": "",
					"Vector": ""
				}
			}
		},
		{
			"ordinal": "6",
			"xmlns": "http://www.icasi.org/CVRF/schema/vuln/1.1",
			"notes": {
				"note": {
					"text": "unbound: integer overflow in a size calculation in dnscrypt/dnscrypt.c",
					"title": "Vulnerability Description",
					"type": "General",
					"ordinal": "1",
					"lang": "en"
				}
			},
			"ReleaseDate": "2019-12-11T00:00:00Z",
			"involvements": {
				"involvement": {
					"party": "Vendor",
					"status": "Completed"
				}
			},
			"CVE": "CVE-2019-25038",
			"remediations": {
				"remediation": {
					"type": "Vendor Fix",
					"description": {
						"text": "\nFor details on how to apply this update, which includes the changes described in this advisory, refer to:\n\nhttps://access.redhat.com/articles/11258\n    ",
						"lang": "en"
					},
					"URL": "https://access.redhat.com/errata/RHSA-2021:1853"
				}
			},
			"references": {
				"reference": [
					{
						"URL": "https://access.redhat.com/security/cve/CVE-2019-25038",
						"Description": "CVE-2019-25038"
					},
					{
						"URL": "https://bugzilla.redhat.com/show_bug.cgi?id=1954796",
						"Description": "bz#1954796: CVE-2019-25038 unbound: integer overflow in a size calculation in dnscrypt/dnscrypt.c"
					}
				]
			},
			"threats": {
				"threat": {
					"type": "Impact",
					"Description": "Moderate"
				}
			},
			"DiscoveryDate": "2021-04-27T00:00:00Z",
			"productstatuses": {
				"status": {
					"type": "Fixed",
					"ProductID": [
						"AppStream-8.4.0.GA:unbound-0:1.7.3-15.el8"
					]
				}
			},
			"acknowledgments": {
				"acknowledgment": {
					"Description": ""
				}
			},
			"cvssscoresets": {
				"scoreset": {
					"BaseScore": "",
					"Vector": ""
				}
			}
		},
		{
			"ordinal": "7",
			"xmlns": "http://www.icasi.org/CVRF/schema/vuln/1.1",
			"notes": {
				"note": {
					"text": "unbound: integer overflow in a size calculation in respip/respip.c",
					"title": "Vulnerability Description",
					"type": "General",
					"ordinal": "1",
					"lang": "en"
				}
			},
			"ReleaseDate": "2019-12-11T00:00:00Z",
			"involvements": {
				"involvement": {
					"party": "Vendor",
					"status": "Completed"
				}
			},
			"CVE": "CVE-2019-25039",
			"remediations": {
				"remediation": {
					"type": "Vendor Fix",
					"description": {
						"text": "\nFor details on how to apply this update, which includes the changes described in this advisory, refer to:\n\nhttps://access.redhat.com/articles/11258\n    ",
						"lang": "en"
					},
					"URL": "https://access.redhat.com/errata/RHSA-2021:1853"
				}
			},
			"references": {
				"reference": [
					{
						"URL": "https://access.redhat.com/security/cve/CVE-2019-25039",
						"Description": "CVE-2019-25039"
					},
					{
						"URL": "https://bugzilla.redhat.com/show_bug.cgi?id=1954797",
						"Description": "bz#1954797: CVE-2019-25039 unbound: integer overflow in a size calculation in respip/respip.c"
					}
				]
			},
			"threats": {
				"threat": {
					"type": "Impact",
					"Description": "Moderate"
				}
			},
			"DiscoveryDate": "2021-04-27T00:00:00Z",
			"productstatuses": {
				"status": {
					"type": "Fixed",
					"ProductID": [
						"AppStream-8.4.0.GA:unbound-0:1.7.3-15.el8"
					]
				}
			},
			"acknowledgments": {
				"acknowledgment": {
					"Description": ""
				}
			},
			"cvssscoresets": {
				"scoreset": {
					"BaseScore": "",
					"Vector": ""
				}
			}
		},
		{
			"ordinal": "8",
			"xmlns": "http://www.icasi.org/CVRF/schema/vuln/1.1",
			"notes": {
				"note": {
					"text": "unbound: infinite loop via a compressed name in dname_pkt_copy",
					"title": "Vulnerability Description",
					"type": "General",
					"ordinal": "1",
					"lang": "en"
				}
			},
			"ReleaseDate": "2019-12-11T00:00:00Z",
			"involvements": {
				"involvement": {
					"party": "Vendor",
					"status": "Completed"
				}
			},
			"CVE": "CVE-2019-25040",
			"remediations": {
				"remediation": {
					"type": "Vendor Fix",
					"description": {
						"text": "\nFor details on how to apply this update, which includes the changes described in this advisory, refer to:\n\nhttps://access.redhat.com/articles/11258\n    ",
						"lang": "en"
					},
					"URL": "https://access.redhat.com/errata/RHSA-2021:1853"
				}
			},
			"references": {
				"reference": [
					{
						"URL": "https://access.redhat.com/security/cve/CVE-2019-25040",
						"Description": "CVE-2019-25040"
					},
					{
						"URL": "https://bugzilla.redhat.com/show_bug.cgi?id=1954799",
						"Description": "bz#1954799: CVE-2019-25040 unbound: infinite loop via a compressed name in dname_pkt_copy"
					}
				]
			},
			"threats": {
				"threat": {
					"type": "Impact",
					"Description": "Moderate"
				}
			},
			"DiscoveryDate": "2021-04-27T00:00:00Z",
			"productstatuses": {
				"status": {
					"type": "Fixed",
					"ProductID": [
						"AppStream-8.4.0.GA:unbound-0:1.7.3-15.el8"
					]
				}
			},
			"acknowledgments": {
				"acknowledgment": {
					"Description": ""
				}
			},
			"cvssscoresets": {
				"scoreset": {
					"BaseScore": "",
					"Vector": ""
				}
			}
		},
		{
			"ordinal": "9",
			"xmlns": "http://www.icasi.org/CVRF/schema/vuln/1.1",
			"notes": {
				"note": {
					"text": "unbound: assertion failure via a compressed name in dname_pkt_copy",
					"title": "Vulnerability Description",
					"type": "General",
					"ordinal": "1",
					"lang": "en"
				}
			},
			"ReleaseDate": "2019-12-11T00:00:00Z",
			"involvements": {
				"involvement": {
					"party": "Vendor",
					"status": "Completed"
				}
			},
			"CVE": "CVE-2019-25041",
			"remediations": {
				"remediation": {
					"type": "Vendor Fix",
					"description": {
						"text": "\nFor details on how to apply this update, which includes the changes described in this advisory, refer to:\n\nhttps://access.redhat.com/articles/11258\n    ",
						"lang": "en"
					},
					"URL": "https://access.redhat.com/errata/RHSA-2021:1853"
				}
			},
			"references": {
				"reference": [
					{
						"URL": "https://access.redhat.com/security/cve/CVE-2019-25041",
						"Description": "CVE-2019-25041"
					},
					{
						"URL": "https://bugzilla.redhat.com/show_bug.cgi?id=1954801",
						"Description": "bz#1954801: CVE-2019-25041 unbound: assertion failure via a compressed name in dname_pkt_copy"
					}
				]
			},
			"threats": {
				"threat": {
					"type": "Impact",
					"Description": "Moderate"
				}
			},
			"DiscoveryDate": "2021-04-27T00:00:00Z",
			"productstatuses": {
				"status": {
					"type": "Fixed",
					"ProductID": [
						"AppStream-8.4.0.GA:unbound-0:1.7.3-15.el8"
					]
				}
			},
			"acknowledgments": {
				"acknowledgment": {
					"Description": ""
				}
			},
			"cvssscoresets": {
				"scoreset": {
					"BaseScore": "",
					"Vector": ""
				}
			}
		},
		{
			"ordinal": "10",
			"xmlns": "http://www.icasi.org/CVRF/schema/vuln/1.1",
			"notes": {
				"note": {
					"text": "unbound: out-of-bounds write via a compressed name in rdata_copy",
					"title": "Vulnerability Description",
					"type": "General",
					"ordinal": "1",
					"lang": "en"
				}
			},
			"ReleaseDate": "2019-12-11T00:00:00Z",
			"involvements": {
				"involvement": {
					"party": "Vendor",
					"status": "Completed"
				}
			},
			"CVE": "CVE-2019-25042",
			"remediations": {
				"remediation": {
					"type": "Vendor Fix",
					"description": {
						"text": "\nFor details on how to apply this update, which includes the changes described in this advisory, refer to:\n\nhttps://access.redhat.com/articles/11258\n    ",
						"lang": "en"
					},
					"URL": "https://access.redhat.com/errata/RHSA-2021:1853"
				}
			},
			"references": {
				"reference": [
					{
						"URL": "https://access.redhat.com/security/cve/CVE-2019-25042",
						"Description": "CVE-2019-25042"
					},
					{
						"URL": "https://bugzilla.redhat.com/show_bug.cgi?id=1954804",
						"Description": "bz#1954804: CVE-2019-25042 unbound: out-of-bounds write via a compressed name in rdata_copy"
					}
				]
			},
			"threats": {
				"threat": {
					"type": "Impact",
					"Description": "Moderate"
				}
			},
			"DiscoveryDate": "2021-04-27T00:00:00Z",
			"productstatuses": {
				"status": {
					"type": "Fixed",
					"ProductID": [
						"AppStream-8.4.0.GA:unbound-0:1.7.3-15.el8"
					]
				}
			},
			"acknowledgments": {
				"acknowledgment": {
					"Description": ""
				}
			},
			"cvssscoresets": {
				"scoreset": {
					"BaseScore": "",
					"Vector": ""
				}
			}
		},
		{
			"ordinal": "11",
			"xmlns": "http://www.icasi.org/CVRF/schema/vuln/1.1",
			"notes": {
				"note": {
					"text": "unbound: symbolic link traversal when writing PID file",
					"title": "Vulnerability Description",
					"type": "General",
					"ordinal": "1",
					"lang": "en"
				}
			},
			"ReleaseDate": "2020-09-09T00:00:00Z",
			"involvements": {
				"involvement": {
					"party": "Vendor",
					"status": "Completed"
				}
			},
			"CVE": "CVE-2020-28935",
			"remediations": {
				"remediation": {
					"type": "Vendor Fix",
					"description": {
						"text": "\nFor details on how to apply this update, which includes the changes described in this advisory, refer to:\n\nhttps://access.redhat.com/articles/11258\n    ",
						"lang": "en"
					},
					"URL": "https://access.redhat.com/errata/RHSA-2021:1853"
				}
			},
			"references": {
				"reference": [
					{
						"URL": "https://access.redhat.com/security/cve/CVE-2020-28935",
						"Description": "CVE-2020-28935"
					},
					{
						"URL": "https://bugzilla.redhat.com/show_bug.cgi?id=1878761",
						"Description": "bz#1878761: CVE-2020-28935 unbound: symbolic link traversal when writing PID file"
					}
				]
			},
			"threats": {
				"threat": {
					"type": "Impact",
					"Description": "Low"
				}
			},
			"DiscoveryDate": "2020-09-11T00:00:00Z",
			"productstatuses": {
				"status": {
					"type": "Fixed",
					"ProductID": [
						"AppStream-8.4.0.GA:unbound-0:1.7.3-15.el8"
					]
				}
			},
			"acknowledgments": {
				"acknowledgment": {
					"Description": "This issue was discovered by Mason Loring Bliss (Red Hat)."
				}
			},
			"cvssscoresets": {
				"scoreset": {
					"BaseScore": "",
					"Vector": ""
				}
			}
		}
	],
	"producttree": {
		"xmlns": "http://www.icasi.org/CVRF/schema/prod/1.1",
		"branch": [
			{
				"type": "Product Family",
				"name": "Red Hat Enterprise Linux",
				"branch": [
					{
						"type": "Product Name",
						"name": "Red Hat Enterprise Linux AppStream (v. 8)",
						"fullproductname": {
							"text": "Red Hat Enterprise Linux AppStream (v. 8)",
							"productid": "AppStream-8.4.0.GA",
							"cpe": "cpe:/a:redhat:enterprise_linux:8::appstream"
						}
					}
				],
				"fullproductname": {}
			},
			{
				"type": "Product Version",
				"name": "unbound-0:1.7.3-15.el8",
				"fullproductname": {
					"text": "unbound-1.7.3-15.el8.src.rpm",
					"productid": "unbound-0:1.7.3-15.el8",
					"cpe": "cpe:/a:redhat:enterprise_linux:8::appstream"
				}
			}
		],
		"relationship": [
			{
				"productreference": "unbound-0:1.7.3-15.el8",
				"relationtype": "Default Component Of",
				"relatestoproductreference": "AppStream-8.4.0.GA",
				"fullproductname": {
					"text": "unbound-0:1.7.3-15.el8 as a component of Red Hat Enterprise Linux AppStream (v. 8)",
					"productid": "AppStream-8.4.0.GA:unbound-0:1.7.3-15.el8",
					"cpe": "cpe:/a:redhat:enterprise_linux:8::appstream"
				}
			}
		]
	}
}
